pulpcore/cli/deb/locale/de/LC_MESSAGES/messages.po

```diff
@@ -0,0 +1,43 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the PACKAGE package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: \n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2021-11-06 21:46+0100\n"
+"PO-Revision-Date: 2021-11-06 21:47+0100\n"
+"Language: de\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"X-Generator: Poedit 3.0\n"
+
+#: pulpcore/cli/deb/context.py:14
+msgid "apt remote"
+msgstr ""
+
+#: pulpcore/cli/deb/context.py:15
+msgid "apt remotes"
+msgstr ""
+
+#: pulpcore/cli/deb/context.py:32
+msgid "apt repository"
+msgstr "APT Repository"
+
+#: pulpcore/cli/deb/context.py:33
+msgid "apt repositories"
+msgstr "APT Repositories"
+
+#: pulpcore/cli/deb/repository.py:107
+#, python-brace-format
+msgid ""
+"Repository '{name}' does not have a default remote. Please specify with '--"
+"remote'."
+msgstr ""
+"Repository '{name}' hat keine voreingestellte Gegenstelle. Bitte mit '--"
```

Comment:
"Remote" is one of our Pulp terms that I am hesitant to translate.
In addition, according to Duden "remote" is already a German word (at least as an adjective), so I would just not translate it here.
See: https://www.duden.de/rechtschreibung/remote
```suggestion
"Repository '{name}' hat keinen voreingestellten Remote. Bitte mit '--"
```